build.gradle

```diff
@@ -20,6 +20,7 @@ sourceCompatibility = '1.8'
 repositories {
     mavenLocal()
     mavenCentral()
+    maven { url 'https://jitpack.io' }
```

Comment:
This can be removed after a release gets made using <https://github.com/GTNewHorizons/SpecialSource/pull/1>

Reply:
Resolved with a new commit